### PR TITLE
Add multi-language devcontainer configurations

### PR DESCRIPTION
**Pull request by louislowjk**
Screenshot 2024-07-31 at 12 14 47 PM

### CODE REVIEW

File: .devcontainer/java/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            ]
+        }
+    },
+    "features": {
+        "ghcr.io/devcontainers/features/java:1": {
+            "version": "17",
+            "installMaven": "true"
+        }
+    },
+    "postCreateCommand": "cd /workspaces/${localWorkspaceFolderBasename}/server && cp .env.example java/.env && cd java && mvn clean install",
```
Comment (rekhaabaidipati) [inline .devcontainer/java/devcontainer.json:50]: Can you please remove this postCreateCommand and test

Comment (rekhaabaidipati) [reply .devcontainer/java/devcontainer.json]: Fixed and resolved

**Review by cnallam — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -57,3 +57,6 @@ Fastlane Flexible Integration allows you to customize and style your payment pag
 #### Key Features
 - Further customize the behavior and experience of your checkout
 - Data Security: Flexible Integration is PCI DSS compliant, ensuring that customer payment information is handled securely
+
+## Codespaces
```
Comment (rekhaabaidipati) [inline README.md:61]: @louislowjk  can you update this readme with content similar to this 
https://github.com/paypal-examples/googlepay?tab=readme-ov-file#paypal-codespaces 

Please add all the content under the PayPal codespaces header and modify the codespaces link as peer this repo. 

This was approved by our content team so wanted to maintain the same for consistency. thanks

**Review by rekhaabaidipati — CHANGES_REQUESTED** [review]
Kindly update readme as per my comment. Rest all looks good

File: README.md (hunk 2 above)
Comment (louislowjk) [reply README.md]: Updated.

**Review by gregjopa — APPROVED** [review]

**Review by allen-tong — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -57,3 +57,20 @@ Fastlane Flexible Integration allows you to customize and style your payment pag
...
 - Further customize the behavior and experience of your checkout
 - Data Security: Flexible Integration is PCI DSS compliant, ensuring that customer payment information is handled securely
+
+# PayPal Codespaces
+
+PayPal codespaces require a client ID and client secret for your app.
+
+### Link to codespaces 
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/louislowjk/fastlane-sample-application)
```
Comment (gregjopa) [inline README.md:67]: @louislowjk what do you think about adding a table to the readme that has a specific link to each codespace which pre-selects the server-side programming language? Something like:


Screenshot 2024-08-06 at 12 24 25 PM

Similar to what we have here: https://github.com/paypal-examples/docs-examples?tab=readme-ov-file#link-to-codespaces

Comment (rekhaabaidipati) [reply README.md:67]: That's a great idea Greg. We will include this in next PR while enabling client side programming languages also. As we are also thinking of a strategy on whether to provide an option to select both client and server side languages.

**Pull request merged**